doc/api/console.markdown

```diff
@@ -2,29 +2,79 @@
 
     Stability: 2 - Stable
 
-The module defines a `Console` class and exports a `console` object.
+The `console` module provides a simple debugging console that is similar to the
```

Comment:
It might be useful to say "similar, but not identical to" as we get a decent number of requests for console features that browsers have.